### PR TITLE
bypass MongoDB; forcibly remove MongoDB's 256MB prealloc files

### PR DESCRIPTION
**Pull request by holta**

### CODE REVIEW

**Comment by holta** [comment]
Attempts to resolve #254 ("what's causing 801MB /var/lib/mongodb ?") by brute force, since the more elegant approaches of the past 5 weeks have so far failed.

**Comment by holta** [comment]
Smoke-tested & functionality-tested.

Please review so we can hopefully merge during Thursday's 10AM EDT call (http://minutes.iiab.io)

(If there are MongoDB experts out there who can in future solve this prealloc file pollution at the source, even better!)

File: roles/mongodb/tasks/main.yml [hunk 1]
```diff
@@ -47,6 +45,20 @@
...
+    paths: /var/lib/mongodb/journal
+    patterns: prealloc.*
+  register: files_to_delete
+
+- name: delete prealloc files
+  file:
+    path: "{{ item.path }}"
+    state: absent
+  with_items: "{{ files_to_delete.files }}"
+
```
Comment (jvonau) [inline roles/mongodb/tasks/main.yml:61]: not mongodb_enabled

**Comment by holta** [comment]
In the long struggle to bring #193 under control (Sugarizer) should the 5 lines below perhaps be commented out and/or any similar experiment to dump MongoDB entirely on a trial basis?  (For a much-more lightweight Sugar without Journal etc)

1. Specifically [roles/1-prep/tasks/computed_vars.yml#L63-L67](https://github.com/iiab/iiab/blob/master/roles/1-prep/tasks/computed_vars.yml#L63-L67) pasted in here:
```
- name: Turn on mongodb if sugarizer enabled
  set_fact:
    mongodb_install: True
    mongodb_enabled: True
  when: sugarizer_enabled
```
...which overrides [roles/mongodb/defaults/main.yml](https://github.com/iiab/iiab/blob/master/roles/mongodb/defaults/main.yml) here:
```
mongodb_install: False
mongodb_enabled: False
```
**_Or perhaps that's not sufficient given...?_**

2. [roles/sugarizer/meta/main.yml](https://github.com/iiab/iiab/blob/master/roles/sugarizer/meta/main.yml) which currently appears as follows:
```
dependencies:
- { role: mongodb, tags: ['generic','mongodb'], when: sugarizer_install }
```
(Should both lines above be commented out, or is commenting out Line 2 above sufficient?)

3. [roles/sugarizer/templates/sugarizer.service.j2](https://github.com/iiab/iiab/blob/master/roles/sugarizer/templates/sugarizer.service.j2) begins with these 3 lines:
```
[unit]
Description=Node.js Sugarizer Server
Requires=After=mongodb.service       # Requires the mongodb service to run first
...
```
(Can line 3 be commented out?)

**Comment by holta** [comment]
Smoke-tested & functionality-tested.

**Pull request merged**